bool colorDetected
color_detection/points[] segments 
